source/changelog.txt

```diff
@@ -12,6 +12,16 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.2.9
+------
+
+*Released June 14, 2024*
+
+``mongosh`` now uses version 6.5.0 of the :driver:`Node.js driver </node/>`.
```

Comment:
Should this be 6.7.0? There is already a RN from v2.20 that states mongosh is using 6.5.0: 
https://preview-mongodbsaraholsonmongodb.gatsbyjs.io/mongodb-shell/DOCSP-40374/changelog/#v2.2.0.

I also dont see any notice of the driver bump and the latest release page: https://github.com/mongodb-js/mongosh/releases/tag/v2.2.9

```suggestion
``mongosh`` now uses version 6.7.0 of the :driver:`Node.js driver </node/>`.
```

Reply:
Yes :)

Reply:
Thanks a ton @ianf-mongodb! Updated!

Reply:
I think it makes sense to omit the Jira release notes then. As Anna mentioned, they don't contain customer facing changes, and as Ian mentioned, they don't contain the node.js driver update either.

Reply:
Just to be clear, what I meant was if we publish these RN's, the Github release page should match the driver bullet. For example for version 1.8 both [the RN's ](https://github.com/mongodb-js/mongosh/releases/tag/v1.8.0)and the [release page](https://github.com/mongodb-js/mongosh/releases/tag/v1.8.0) mention the driver bump. Would it be possible to add this to the release page @addaleax?

Reply:
@ianf-mongodb So ... in an ideal world we'd have this, but at least as long as we don't manually curate release notes for Github, this isn't going to show up automatically. In this case, we've had two failed releases (hence skipping versions 2.2.7 and 2.2.8), and for the change that bumped the driver version, the engineer creating it marked it as a "no direct user impact" change (which is, for the most part, accurate – it's a dependency bump, not an API change).

We can do a one-off and edit the GH release notes to be more expressive here, but for mongosh, we haven't really done this; if you think that it's helpful for us to do this, then we should probably integrate this into our release process as well.

Reply:
So – for a bit more context, I already mentioned this to Sarah on Slack, but TSEs have historically advocated specifically for logging Node.js driver version updates in changelogs because that way, they would be able to identify which driver version mongosh is using when a user reports a bug (and only specifies the mongosh version, which is reasonable for a mongosh user) and then potentially try to figure out whether the issue originated from the driver or not. That's why this PR was created in this form in the first place, rather than just saying "no impact to users", even though it _should_ have no impact to users.

Reply:
LGTM pending we update the process for mongosh release notes to reflect this workflow on: https://wiki.corp.mongodb.com/display/DE/mongosh+Release+Notes